frontend/src/app/main/ui/browser_hooks.cljs

```diff
@@ -0,0 +1,31 @@
+(ns app.main.ui.browser-hooks
```

Comment:
After the broadcast and storage changes, I think thare are no reason to have it separated, I'm wrong?